deploy/helm/sifnode/templates/sifnode-deployment.yaml

```diff
@@ -71,8 +71,6 @@ spec:
             mountPath: /root
       volumes:
       - name: root
```

Comment:
@intl-man if we want to remove the claim we need to remove the name of the claim volume as well.